packages/web/app/src/components/target/settings/registry-access-token.tsx

```diff
@@ -293,11 +236,8 @@ export function GenerateTokenContent(props: {
                     dataCy="registry-access-scope"
                     key={props.selectedScope}
                     scope={RegistryAccessScope}
-                    canManageScope={
-                      props.manager.canAccessTarget(RegistryAccessScope.mapping['read-only']) ||
-                      props.manager.canAccessTarget(RegistryAccessScope.mapping['read-write'])
-                    }
-                    checkAccess={props.manager.canAccessTarget}
+                    canManageScope={true}
+                    checkAccess={() => true}
```

Comment:
Within our new permission system we only have `targetAccessToken:modify` and no more `read-only` and `read-write` distinction.

Currenlty, we still have an assertion within `Mutation.createToken`, to check for the assigned scopes.

In the future we will remove this assertion as we decided that someone with `targetAccessToken:modify` permissions always can modify target access tokens.